### PR TITLE
Fixing withBackgrounds on config.js

### PR DESCRIPTION
**Pull request by gabrielbs**
Issue:

## What I did
Fixed the example of how add backgrounds on all stories

## How to test
Try import withBackgrounds as an object

Is this testable with Jest or Chromatic screenshots? No
Does this need a new example in the kitchen sink apps? No
Does this need an update to the documentation? Yes

If your answer is yes to any of these, please make sure to include it in your PR.

For maintainers only: Please tag your pull request with at least one of the following:
`["cleanup", "BREAKING CHANGE", "feature request", "bug", "documentation", "maintenance", "dependencies", "other"]`

### CODE REVIEW

File: addons/backgrounds/README.md [hunk 1]
```diff
@@ -45,7 +45,7 @@ You can add the backgrounds to all stories with `addDecorator` in `.storybook/co
 
 ```js
 import { addDecorator } from '@storybook/react'; // <- or your storybook framework
-import { withBackgrounds } from '@storybook/addon-backgrounds';
+import withBackgrounds from '@storybook/addon-backgrounds';
```
Comment (igor-dv) [inline addons/backgrounds/README.md]: Default export is deprecated. Why would you like to add it to README?

Comment (gabrielbs) [reply addons/backgrounds/README.md]: I meant that the correct way to import the component is without the curly brackets. With the curly brackets are returning undefined

Comment (igor-dv) [reply addons/backgrounds/README.md]: What version are you using?

Comment (gabrielbs) [reply addons/backgrounds/README.md]: The version is ^3.4.10

Comment (igor-dv) [reply addons/backgrounds/README.md]: And the README you are changing is for v4.alpha

Comment (gabrielbs) [reply addons/backgrounds/README.md]: The installation instructions of this file is installing the version 3.4.10. 
I think that accept this change while the v4 is not on npm will prevent mistakes

**Comment by igor-dv** [comment]
@shilman, can we merge this only to the v3.4 branch?

**Comment by ndelangen** [comment]
We need to rebase this to merge to 3.4 branch

**Pull request closed without merging**